koku/masu/test/external/downloader/azure_local/test_azure_local_report_downloader.py

```diff
@@ -101,7 +101,9 @@ def test_download_file(self):
         self.assertEqual(etag, second_run_etag)
         self.assertEqual(full_file_path, expected_full_path)
 
-    def test_download_report(self):
+    @patch("masu.external.downloader.azure_local.azure_local_report_downloader.remove_files_not_in_set_from_s3_bucket")
+    @patch("masu.util.aws.common.copy_data_to_s3_bucket")
+    def test_download_report(self, _, __):
```

Comment:
Same comment as above. If we can, `*args` looks less surprising to the reader.
```suggestion
    def test_download_report(self, *args):
```